Assets/Scripts/Gameplay/UI/UI.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class UI : MonoBehaviour
```

Comment:
UI is not specific enough for a classname.
The class should also be in the ProjectWendigo namespace.